spec/features/courses/subjects/edit_spec.rb

```diff
@@ -30,29 +30,45 @@
     stub_api_v2_resource(course, include: "subjects,sites,provider.sites,accrediting_provider")
   end
 
-  scenario "can visit the subject edit page" do
-    course_details_page.load_with_course(course)
-    expect(course_details_page).to be_displayed
-    course_details_page.edit_subjects_link.click
-    expect(subjects_page).to be_displayed
+  context "with given subjects" do
```

Comment:
Not sure what the difference is between this `"with given subjects"` context, and the `"with a given set if subjects"` context below? And it seems the way `course` is built here is essentially identical  to the one on line 14, so is this unnecessary extra noise?